math_short.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+    <head></head>
+    <body>
+        <script>
+            const firstPrompt = prompt("Please, enter first number");
+            const secondPrompt = prompt("Please, enter second number");
+            const num1 = +firstPrompt;
+            const num2 = +secondPrompt;
+            alert(firstPrompt+"+"+secondPrompt+"="+ +(num1+num2)+"\n"+
```

Comment:
Reformat a bit, and add spaces between variables and operators, such code will be more readable.

```
const message = (
  firstPrompt + " + " + secondPrompt + " = " + (num1 + num2) + "\n" +
  firstPrompt + " - " + secondPrompt + " = " + (num1 - num2) + "\n" +
  firstPrompt + " * " + secondPrompt + " = " + (num1 * num2) + "\n" +
  firstPrompt + " / " + secondPrompt + " = " + (num1 / num2)
);

alert(message);
```